services/order.py

```diff
@@ -0,0 +1,33 @@
+from datetime import datetime
+
+from django.db import transaction
+from django.db.models import QuerySet
+
+from db.models import Order, User, Ticket
+
+
+@transaction.atomic
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: str = None
+) -> Order:
+    user = User.objects.get(username=username)

```

Comment:
You should use `get_user_model` here instead of importing and using the `User` model directly.